### PR TITLE
Developer branch

### PR DESCRIPTION
**Pull request by mcastorena0316**
I think im ready for my review. Thanks guys!

### CODE REVIEW

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @mcastorena0316,

### Great job thus far,:raised_hands: just a few suggestions to wrap it up:

- [ ] There is a `SlackSetup.exe` file inside the images folder, let's get rid of this.

- [ ] Why do you need `package.json` or `babel`? I'd love to know.

- [ ] Let's get rid of the horizontal scrollbar, this shows that there is an overflow somewhere in your design.

- [ ] In the mobile view, `More Stories` should come immediately after `Top Story`, then followed by `Culture & Travel`, and `Featured Stories` before `Opinion`. Let's fix this order, you can use the `order` class of bootstrap for this. In summary, be sure to order the sections the same way it is ordered on the Newsweek website without affecting the medium or large screen ordering.

**Kindly submit for another code review once these issues have been fixed.** _Happy coding! :tada:_

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,2988 @@
...
+  "requires": true,
+  "lockfileVersion": 1,
+  "dependencies": {
+    "@babel/code-frame": {
+      "version": "7.5.5",
+      "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.5.5.tgz",
+      "integrity": "sha512-27d4lZoomVyo51VegxI20xZPuSHusqbQag/ztrBC7wegWoQ1nLREPVSKSW8byhTlzTKyNE4ifaTA6lCp7JjpFw==",
+      "requires": {
+        "@babel/highlight": "^7.0.0"
+      }
```
Comment (suretrust) [inline package-lock.json]: Could you tell me why you need babel?

File: style.css [hunk 2]
```diff
@@ -0,0 +1,926 @@
...
+}
+
+.image-text-small::after {
+  display: block;
+  clear: both;
+  content: "";
+}
+
+.image-small {
+  /* float: left; */
```
Comment (suretrust) [inline style.css:296]: Let's remove the commented code.

**Review by suretrust — CHANGES_REQUESTED** [review]

**Comment by mcastorena0316** [comment]
Requiring a new review please.

**Review by imhta — APPROVED** [review]
# Project Approved 👏 
Hello @mcastorena0316, Congratulations you have done great job. 






----
#### Don't forgot to sumbit project completion form
 **Pon Muthu Selvam N**
Feel free contact me 
[Github //@imhta](https://github.com/imhta) | [Twitter //@nponmuthuselvam](https://twitter.com/ponmuthuselvam)

**Pull request merged**